__tests__/commands/openapi/__snapshots__/inspect.test.ts.snap

```diff
@@ -93,13 +103,8 @@ additionalProperties:
  · #/components/schemas/ParameterizedHeader/properties/parameters/additionalProperties
 
 circularRefs:
- · #/components/schemas/BodyPart/properties/parent
```

Comment:
These snapshots got updated because of the circular reference performance change in `@apidevtools/json-schema-ref-parser`.